frontend/javascript/components/redaction/pdf-redaction.vue

```diff
@@ -149,6 +150,7 @@
 
         <div class="btn-group me-1 toolbar-modes py-2">
           <button
+            type="button"
```

Comment:
Let's maybe switch to a [checkbox toggle button](https://getbootstrap.com/docs/5.3/forms/checks-radios/#checkbox-toggle-buttons) here as well